-tags.tf

```diff
@@ -0,0 +1,5 @@
+locals {
+  common_tags = merge(var.tags, {
```

Comment:
did you mean to open a PR against your own fork?

Reply:
No...Oops.